test/fakeintake/client/client.go

```diff
@@ -1,3 +1,7 @@
+// APIFakeIntakeRouteStatsGETResponse exported type should have comment or be unexported
+// RouteStat exported type should have comment or be unexported
+// APIFakeIntakePayloadsGETResponse exported type should have comment or be unexported
+// Payload exported type should have comment or be unexported
```

Comment:
Why these changes?

Reply:
good catch this is a corner case of my script I have missed